flexmeasures/ui/templates/views/sensors.html

```diff
@@ -25,40 +25,154 @@
+
+                                // Show the spinner
+                                spinner.classList.remove('d-none');
+
+                                fetch('/api/v3_0/sensors/' + {{ sensor.id }} + '/stats')
+                                    .then(response => response.json())
+                                    .then(data => {
+                                        // Remove 'status' sourceKey
+                                        delete data['status'];
+
+                                        if (Object.keys(data).length > 0) {
+                                            // Show the header and dropdown container
+                                            statsHeader.classList.remove('d-none');
+                                            dropdownContainer.classList.remove('d-none');
+
+                                            // Populate the dropdown menu with sourceKeys
+                                            Object.keys(data).forEach(sourceKey => {
+                                                const dropdownItem = document.createElement('li');
+                                                const dropdownLink = document.createElement('a');
+                                                dropdownLink.className = 'dropdown-item';
+                                                dropdownLink.href = '#';
+                                                dropdownLink.textContent = sourceKey;
+                                                dropdownLink.dataset.sourceKey = sourceKey;
+
+                                                dropdownLink.addEventListener('click', function(event) {
+                                                    event.preventDefault();
+                                                    const selectedSourceKey = event.target.dataset.sourceKey;
+                                                    dropdownButton.textContent = selectedSourceKey;
+                                                    updateTable(data[selectedSourceKey]);
+                                                });
+
+                                                dropdownItem.appendChild(dropdownLink);
+                                                dropdownMenu.appendChild(dropdownItem);
+                                            });
+
+                                            // Update the table with the first sourceKey's data by default
+                                            const firstSourceKey = Object.keys(data)[0];
+                                            dropdownButton.textContent = firstSourceKey;
+                                            updateTable(data[firstSourceKey]);
+                                        } else {
+                                            // If the stats table is empty, make the properties table full width
+                                            propertiesContainer.classList.remove('col-sm-6');
```

Comment:
Why do we make this layout change? Seems weird.

Reply:
In case the sensor does not have data then the `stats` table will not be shown and the `properties` table will be on the top left of the sensor chart with an empty space to it's right. So, I wanted the `properties` table would cover the empty space on the top right of the chart.

Reply:
You can see the empty space.

![image](https://github.com/user-attachments/assets/3c7ede0a-d712-4f88-98e2-6cc05a42877e)